src/main.cpp

```diff
@@ -780,6 +780,10 @@ Use Ctrl+C to stop.)");
   -m, --muxer               Set the output format to a specific muxer instead of detecting it
                             from the filename.
 
+  -s, --show-mouse          Shows the mouse in the output file. This is the default behavior.
```

Comment:
I don't think we need both options: the way it is now, `-s` has no effect (because it is the default).

Reply:
This is done for consistency. If you can hide your mouse, you should also be able to show it.